create-cluster-secrets.sh

```diff
@@ -48,7 +46,7 @@ do
     done
     kubectl -n ${CLUSTER_NAME} create secret generic node${i}-validators $files --dry-run=client -o yaml | kubectl apply -f -
     kubectl -n ${CLUSTER_NAME} create secret generic node${i}-charon-enr-private-key --from-file=charon-enr-private-key=./.charon/${CLUSTER_NAME}/node${i}/charon-enr-private-key --dry-run=client -o yaml | kubectl apply -f -
-    kubectl -n ${CLUSTER_NAME} create secret generic node${i}-cluster-lock --from-file=cluster-lock.json=./.charon/${CLUSTER_NAME}/cluster-lock.json --dry-run=client -o yaml | kubectl apply -f -
+    kubectl -n ${CLUSTER_NAME} create secret generic node${i}-cluster-lock --from-file=cluster-lock.json=./.charon/${CLUSTER_NAME}/node${i}/cluster-lock.json --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
I don't think that this would work for canary clusters. The way we have it set up is currently: 
```
canary-goerli-1/
├── canary-goerli-1.env
├── cluster-lock.json
└── node0/
    ├── charon-enr-private-key
    └── validator_keys/
        ├── keystore-0.json
        └── keystore-0.txt
```